run/all_native.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
+import json
+from multiprocessing import cpu_count
+from numpy import linspace
+from subprocess import check_output
+import os
+
+# Experiment information
+IMAGE_NAME = "experiment-covid-native"
+PROJ_ROOT = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+BASE_DIR = "{}/../..".format(PROJ_ROOT)
+with open(os.path.join(BASE_DIR, "VERSION")) as version_file:
+    VERSION = version_file.read().strip()
+RESULTS_FILE = "{}/results/covid/covid_native.dat".format(BASE_DIR)
+
+# Benchmark parameters
```

Comment:
These are the parameters that I mention could (should) eventually be moved elsewhere.